### PR TITLE
Avm2: More class stubs

### PR DESCRIPTION
**Pull request by adrian17**
These are the ones we needed for `you have to burn the rope`.

### CODE REVIEW

File: core/src/avm2/globals/flash/ui/contextmenuitem.rs [hunk 1]
```diff
@@ -0,0 +1,48 @@
+//! `flash.ui.ContextMenuItem` builtin/prototype
```
Comment (torokati44) [inline core/src/avm2/globals/flash/ui/contextmenuitem.rs:1]: In `avm1`, this file is called `context_menu_item.rs`, while in AVM2, there aren't many `_`s in file names... Not sure which variant would be more consistent - maybe it should be kept as is, even though this variation is a bit unfortunate.

**Comment by adrian17** [comment]
> Just stubbing that out to do nothing and return Undefined makes the loop work.

Can do that (though ideally I think they will be slots, otherwise we'd need a separate Rust-layer object enum variant which is silly).

> And z0r 6701 (among others) tries to set the enabled property of a ContextMenuItem instance, but I don't see in the reference where that would be coming from (inherited)...?

Yeah, it's in NativeMenuItem.

File: core/src/avm2/globals/flash/ui/contextmenuitem.rs (hunk 1 above)
Comment (adrian17) [reply core/src/avm2/globals/flash/ui/contextmenuitem.rs:1]: Yeah, by now, most avm2 globals' files have naming convention without `_`. I didn't contest this and just stayed with it so far.
(also, as extra argument, `i_bitmap_drawable.rs` looks kinda weird :) )

Comment (torokati44) [reply core/src/avm2/globals/flash/ui/contextmenuitem.rs:1]: Okay, that's fair... :D

**Comment by torokati44** [comment]
Of the 41 affected z0r.de loops, 40 are now stuck on some other class - which is good! (The majority of them on the related `flash.events.ContextMenuEvent`.)
And z0r 4663 now isn't missing any type, it almost works perfectly, with just a little white flash after each loop, which I hope #5876 might improve upon.

**Review by kmeisthax — APPROVED** [review]

**Pull request merged**